### PR TITLE
Add `useSetThermostatOff`

### PR DESCRIPTION
**Pull request by xplato**

### CODE REVIEW

**Review by razor-x — APPROVED** [review]

File: src/lib/seam/thermostats/use-update-off.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    },
+  })
+}
+
+function getUpdatedThermostat(thermostat: ThermostatDevice): ThermostatDevice {
+  return {
+    ...thermostat,
+    properties: {
+      ...thermostat.properties,
+      is_fan_running: false,
```
Comment (razor-x) [inline src/lib/seam/thermostats/use-update-off.ts:74]: Are we sure this always results in the fan turning off?

image

**Review by razor-x — CHANGES_REQUESTED** [review]
Actually, quick question to resolve before merge

File: src/lib/seam/thermostats/use-update-off.ts
Comment (razor-x) [inline src/lib/seam/thermostats/use-update-off.ts:1]: Need to rename file.

Comment (xplato) [reply src/lib/seam/thermostats/use-update-off.ts:1]: Ooh thank you

**Review by razor-x — APPROVED** [review]

**Pull request merged**